singularity-eos/eos/eos_spiner.hpp

```diff
@@ -475,16 +475,16 @@ class SpinerEOSDependsRhoSie : public EosBase<SpinerEOSDependsRhoSie> {
   }
   PORTABLE_FORCEINLINE_FUNCTION Real rhoMax() const { return rhoMax_; }
   PORTABLE_FORCEINLINE_FUNCTION Real TMin() const {
-    return fromLog_(T_.range(0).min(), lTOffset_);
+    return fromLog_(sie_.range(0).min(), lTOffset_);
   }
   PORTABLE_FORCEINLINE_FUNCTION Real TMax() const {
-    return fromLog_(T_.range(0).max(), lTOffset_);
+    return fromLog_(sie_.range(0).max(), lTOffset_);
   }
   PORTABLE_FORCEINLINE_FUNCTION Real sieMin() const {
-    return fromLog_(sie_.range(0).min(), lEOffset_);
+    return fromLog_(T_.range(0).min(), lEOffset_);
```

Comment:
I still don't get why `sieMin()` isn't just the lowest entry on the energy grid

Reply:
It essentially is. But it depends on which grid. `SpinerEOSDependsRhoSie` has a grid where sie is the independent variable and a grid where it's the dependent variable. This bug fix pulls it from the minimum value where it's an independent variable.

Reply:
I'd assume in the dependent variable case that the minimum observed sie is then at the whims of interpolation resolution near the true minimum and I don't think `|e_min,interp - e_min,true|` is really bounded.